packages/edit-post/src/components/sidebar/post-link/index.js

```diff
@@ -95,7 +96,10 @@ function PostLink( {
 				</div>
 			) }
 			<p className="edit-post-post-link__preview-label">
-				{ __( 'Preview' ) }
+				{ sprintf(
+					/* translators: %s: post type singular name */
+					__( 'View %s' ), postLabel
```

Comment:
This should use the proper `view_item` label for this post type as generated by `get_post_type_labels` in WordPress. Anything else does not translate well.